### DISCUSSION TITLE
Why cannot use 'LicenseRef' license name when initializing the package.json with 'npm init' command?

### DISCUSSION THREAD

**Discussion by dd-jy** [npm/rfcs#563] [RRFC]
Why cannot use 'LicenseRef' license name when initializing the package.json with 'npm init' command?

![image](https://user-images.githubusercontent.com/50347670/160568702-b504f1cc-e26d-4f57-bdf5-d7d2a8189d27.png)

If users wants to add the proprietary license name in package.json, they may also want to add the license name to match the spdx license expression. Because it is easier to manage the license name of their package than using 'SEE LICENSE IN <filename>'.

What about changing to accept 'LicenseRef-' spdx license expression?

**Comment by ljharb** [comment]
I believe this is managed by https://npmjs.com/licensee; you may want to file an issue there.

**Reply by dd-jy** [reply]
Thanks! 👍

**Reply by dd-jy** [reply]
@ljharb 
Do I need to make an issue in https://github.com/npm/cli?

**Reply by darcyclarke** [reply]
No. If you get support adding into `licensee` we will pull it in as a dep update.
